machineLearning/sentimentAnalysis/twitterSentimentAnalysis/twitterSentimentLambdaFunction/twitterSentiment/app.py

```diff
@@ -0,0 +1,482 @@
+import json
+import boto3
+from pymongo import MongoClient
+import snscrape.modules.twitter as sntwitter
+import os
+import pymongo
+import pandas as pd
+import pickle
+import nltk
+import string
+from statistics import mean
+
+### Directory Setup ###
+# For relative imports use the directory where script is running
+# This will change dynamically when running locallly or on AWS Lambda
+dir_path = os.path.dirname(os.path.realpath(__file__))
+nltk.data.path.append(f"{dir_path}/nltk_data")
+from nltk.tokenize import RegexpTokenizer
+from nltk.sentiment import SentimentIntensityAnalyzer
+
+sia = SentimentIntensityAnalyzer()
+
+
+def lambda_handler(event, context):
+    mongoURI = getSecret("MONGO_URI").get("Parameter").get("Value")
+
+    # Create Connection
+    client = getMongoConnection(mongoURI)
+    # Select the Database
+    database = client[os.environ["DATABASENAME"]]
```

Comment:
This will be "test", think its why no tweets where showing up on live site yesterday, should be "dev".

Reply:
yep was using test for testing. This isn't deployed at all so it was probably empty alright. The old lambda is still running so i'd say that's what got added to the db